### PR TITLE
snippet for datastoreio

### PR DESCRIPTION
**Pull request by vikkyrk**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by vikkyrk** [comment]
R: @aaltay @hadarhg

File: sdks/python/apache_beam/examples/snippets/snippets_test.py [hunk 1]
```diff
@@ -470,6 +470,13 @@ def test_model_textio(self):
         ['aa', 'bb', 'bb', 'cc', 'cc', 'cc'],
         self.get_output(result_path, suffix='.csv'))
 
+  def test_model_datastoreio(self):
+    # We cannot test datastoreio functionality in unit tests therefore we limit
+    # ourselves to making sure the pipeline containing Datastore read and write
+    # transforms can be built.
+    # TODO(vikasrk): Expore using Datastore Emulator.
+    self.assertEqual(None, snippets.model_datastoreio())
```
Comment (aaltay) [inline sdks/python/apache_beam/examples/snippets/snippets_test.py]: What are you checking here? Would not it raise an exception if it fails in some way unexpected?

File: sdks/python/apache_beam/examples/snippets/snippets.py [hunk 2]
```diff
@@ -891,6 +891,47 @@ def filter_words(x):
...
+  musicians = p | 'Musicians' >> beam.Create(
+      ['Mozart', 'Chopin', 'Beethoven', 'Bach'])
+
+  def to_entity(content):
+    entity = entity_pb2.Entity()
+    googledatastore.helper.add_key_path(entity.key, kind, str(uuid.uuid4()))
+    googledatastore.helper.add_properties(entity, {'content': unicode(content)})
+    return entity
+
+  entities = musicians | "To Entity" >> beam.Map(to_entity)
```
Comment (aaltay) [inline sdks/python/apache_beam/examples/snippets/snippets.py]: be consistent with quotes. 

(I will look to see if we can enforce this with existing linters.)

File: sdks/python/apache_beam/examples/snippets/snippets_test.py (hunk 1 above)
Comment (vikkyrk) [inline sdks/python/apache_beam/examples/snippets/snippets_test.py]: You are right. I was just referring to https://github.com/apache/incubator-beam/blob/python-sdk/sdks/python/apache_beam/examples/snippets/snippets_test.py#L477.

Fixed both.

**Comment by aaltay** [comment]
LGTM, thank you.

**Comment by vikkyrk** [comment]
R: @robertwb

**Pull request closed without merging**